builder-support/debian/authoritative/debian-buster/pdns-backend-lmdb.dirs

```diff
@@ -0,0 +1 @@
+var/lib/powerdns
```

Comment:
given the fact that the gsqlite3 and bind backends _want_ to use this dir (due to systemd's ProtectSystem setting), would it make sense to declare this dir in the pdns-server package?

Reply:
pdns-backend-bind and pdns-backend-gsqlite3 both have the same dirs file, so they will create it. I don't think we should move that into pdns-server.